widgets-bundle/getting-started/creating-a-widget.md

```diff
@@ -44,45 +44,14 @@ Video URI: http://example.com/hello-world-widget-video
 
 ## Widget class
 
-Now you'll need to create a class which extends the `SiteOrigin_Widget` abstract base class and, as a minimum, overrides the `get_template_name` and `get_style_name` abstract methods. You'll also need to register your widget class with the SiteOrigin Widgets Bundle using the `siteorigin_widget_register` function, passing in the widget id, widget file path, and widget class name as arguments.
+Now you'll need to create a class which extends the `SiteOrigin_Widget` abstract base class. The `SiteOrigin_Widget` class is based on the WordPress Widgets API so a constructor containing information about the widget is required.
```

Comment:
Two small changes here just leaning on Grammarly a bit:

`Now you'll need to create a class that extends the `SiteOrigin_Widget` abstract base class. The `SiteOrigin_Widget` class is based on the WordPress Widgets API, so a constructor containing information about the widget is required.`